File: P2/NapakalakiRuby/pkg/NapakalakiRuby-0.0.1/lib/Prize.rb
class Prize
  attr_reader :treasures, :level #Consultor GET
  
  def initialize(treasures, level)
    @treasures = treasures
    @level = level
  end
  
  def to_s
    "Niveles: #{@level} \n Premio: #{@treasures}"
  end
end

File: P1/Napakalaki/src/napakalaki/Prize.java
package napakalaki;

/**
 *
 * @author Francisco Fernández
 */
public class Prize {
    private int treasures;
    private int level;
    
    public Prize(int treasures, int level){
        this.treasures=treasures;
        this.level=level;
    }
    public int getLevel(){
        return this.level;
    }
    public int getTreasures(){
        return this.treasures;
    }
    public String toString(){
        return "Treasures = " + Integer.toString(treasures) + "Niveles = " + Integer.toString(level);
    }
}
